regtests/client/python/cli/command/namespaces.py

```diff
@@ -91,7 +91,8 @@ def execute(self, api: PolarisDefaultApi) -> None:
         elif self.namespaces_subcommand == Subcommands.DELETE:
             catalog_api.drop_namespace(prefix=self.catalog, namespace=UNIT_SEPARATOR.join(self.namespace))
         elif self.namespaces_subcommand == Subcommands.GET:
-            catalog_api.namespace_exists(prefix=self.catalog, namespace=UNIT_SEPARATOR.join(self.namespace))
-            print(json.dumps({"namespace": '.'.join(self.namespace)}))
+            print(catalog_api.load_namespace_metadata(
+                      prefix=self.catalog,
+                      namespace=UNIT_SEPARATOR.join(self.namespace)).to_json())
```

Comment:
I don't know if we really want to print the unit separator here

Reply:
Here the UNIT_SEPARATOR is actually just for the input argument to the API call; the main change here is for consistency with the other `get` commands, we now just dump the actual full JSON response.

In practice it means the `namespace` field becomes a json array.

I'd say we can eventually have two modes:

1. Dump json responses (current behavior of all the commands)
2. Pretty-print responses - maybe in a table layout or similar, and we can have syntactic sugar of showing dot-separated namespace hierarchies

Reply:
It's not true that we always exactly dump the json responses -- this is most obvious with `list` commands, e.g. for listing catalogs we do:
```
            for catalog in api.list_catalogs().catalogs:
                print(catalog.to_json())
```

Having said that, I didn't realize the response came back as a JSON array. I think that's probably fine, let's just leave it then 👍